From bd0f027b0b511a98a8b2ac1bafdb308e284b3c5a Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
Date: Thu, 18 Nov 2021 14:46:36 +0100
Subject: [PATCH 120/120] configs: update nezha_defconfig

Enable missing configuration items:

* CONFIG_CGROUPS - needed for running systemd
* CONFIG_EFI - needed for booting via UEFI
* CONFIG_VFAT - needed for accessing the ESP

Signed-off-by: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
---
 arch/riscv/configs/nezha_defconfig | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/arch/riscv/configs/nezha_defconfig b/arch/riscv/configs/nezha_defconfig
index 59e76fcadf2e..f774f13bd921 100644
--- a/arch/riscv/configs/nezha_defconfig
+++ b/arch/riscv/configs/nezha_defconfig
@@ -4,6 +4,7 @@ CONFIG_DEFAULT_HOSTNAME="nezha"
 CONFIG_NO_HZ_IDLE=y
 CONFIG_HIGH_RES_TIMERS=y
 CONFIG_PREEMPT=y
+CONFIG_CGROUPS=y
 CONFIG_EXPERT=y
 # CONFIG_SYSFS_SYSCALL is not set
 # CONFIG_VM_EVENT_COUNTERS is not set
@@ -12,8 +13,8 @@ CONFIG_EXPERT=y
 # CONFIG_SLAB_MERGE_DEFAULT is not set
 CONFIG_SOC_SUNXI=y
 # CONFIG_RISCV_ERRATA_ALTERNATIVE is not set
-# CONFIG_RISCV_ISA_C is not set
-# CONFIG_EFI is not set
+CONFIG_RISCV_ISA_C=y
+CONFIG_EFI=y
 CONFIG_PHYS_RAM_BASE=0x40000000
 CONFIG_PM=y
 CONFIG_PM_DEBUG=y
@@ -198,11 +199,14 @@ CONFIG_EXT4_FS=y
 # CONFIG_MANDATORY_FILE_LOCKING is not set
 # CONFIG_DNOTIFY is not set
 CONFIG_FANOTIFY=y
+CONFIG_VFAT_FS=y
 CONFIG_TMPFS=y
 CONFIG_TMPFS_XATTR=y
 CONFIG_TMPFS_INODE64=y
 # CONFIG_MISC_FILESYSTEMS is not set
 # CONFIG_NETWORK_FILESYSTEMS is not set
+CONFIG_NLS_CODEPAGE_437=y
+CONFIG_NLS_ISO8859_1=y
 CONFIG_CRYPTO_USER_API_HASH=y
 CONFIG_CRYPTO_USER_API_SKCIPHER=y
 CONFIG_CRYPTO_USER_API_RNG=y
-- 
2.32.0

